domains/pallets/evm_nonce_tracker/src/lib.rs

```diff
@@ -41,11 +42,55 @@ mod pallet {
     pub(super) type AccountNonce<T: Config> =
         StorageMap<_, Identity, T::AccountId, U256, OptionQuery>;
 
+    /// Storage to hold EVM contract creation allow list accounts.
+    #[pallet::storage]
+    pub(super) type ContractCreationAllowedBy<T: Config> =
+        StorageValue<_, PermissionedActionAllowedBy<T::AccountId>, OptionQuery>;
+
```

Comment:
Would be ideal to make it a value query and return default. Default being Anyone else this will always disallow any contract creation on Taurus EVM

Reply:
Still believe value query makes more sense with default being allowed by anyone. We will end with two variants, Allowed and Allow specific addresses. This will also avoid a migration on taurus evm